frontend/app/views/event/page.scala.html

```diff
@@ -49,7 +49,12 @@ <h1 class="event-masthead__name qa-event-detail-name">@event.name.text</h1>
                     }
                 </div>
             }
-            <img class="event-header__logo" src="@Asset.at("images/providers/" + event.providerOpt.getOrElse(event.metadata.identifier) + ".svg")">
+            @if(event.providerOpt.getOrElse(event.metadata.identifier) == "5x15") {
```

Comment:
I know, I know. Open to better suggestions!

(also, there's another place this SVG logic exists, `seriesDetail.scala.html`, but I don't think it's needed here as the entire thing is wrapped in `@if(event.providerOpt.isEmpty) {`)